### DISCUSSION TITLE
Suppress cups error message - Raspberry pi

### DISCUSSION THREAD

**Discussion by Ulmerju** [OpenPrinting/cups#526] [Q&A]
When printing, I always get an error message which opens in the foreground when the paper or the ink runs out.

similar to this message:
![pasted-from-clipboard](https://user-images.githubusercontent.com/109333500/200927019-127fb227-eeb6-4bed-bccb-ec5e7375a108.png)

I run a photo box program (pibooth) in full screen mode on a Raspberry Pi 4.
This message makes my program, which runs in full-screen mode, smaller. The message opens above the program
I want to stop that...

Is there a way to prevent/intercept this information or its appearance?

Many Thanks
Julian

**Comment by benjaminaigner** [comment]
Dear @Ulmerju ,
just had the same problem and was looking for an answer here. Nevertheless, found something:

If you remove the GUI package, this message is not shown anymore. Please note, that without this package you have to control your printers via the webUI (http://localhost:631):

`sudo apt remove system-config-printer`
